test/src/tests-identities-attributes.ts

```diff
@@ -1134,18 +1232,13 @@ describe('identities and attributes', function() {
             },
         };
 
-        window.mParticle.config.flags = {
```

Comment:
Why is this flag being removed?

Reply:
it looks like it succeeds with and without, so i must have removed it because it was extraneous

Reply:
i'll add it back for now _just_ in case, and these can probably be cleaned up in a future ticket that i will also make